{{cookiecutter.project_slug}}/include/{{cookiecutter.project_slug}}/example.hpp

```diff
@@ -1,6 +1,35 @@
 #pragma once
 
-#include <iostream>
+#include <filesystem>
+
+#include "emp/config/ArgManager.hpp"
+#include "emp/prefab/ConfigPanel.hpp"
+#include "emp/web/UrlParams.hpp"
+#include "emp/web/web.hpp"
+
+#include "{{cookiecutter.project_slug}}/ProjectConfig.hpp"
+
+void AddExistingConfig({{ cookiecutter.project_name.replace(' ', '') }}Config & config, emp::ArgManager & am) {
```

Comment:
I would put config stuff into its own header `config_setup.hpp` and make free functions `snake_case`